tempesta_fw/http.c

```diff
@@ -243,14 +244,14 @@ int
 tfw_http_send_200(TfwHttpReq *req)
 {
 	TfwStr rh = {
-		.ptr = (TfwStr []){
-			{ .ptr = S_200_PART_01, .len = SLEN(S_200_PART_01) },
-			{ .ptr = *this_cpu_ptr(&g_buf), .len = SLEN(S_V_DATE) },
-			{ .ptr = S_200_PART_02, .len = SLEN(S_200_PART_02) },
-			{ .ptr = S_CRLF, .len = SLEN(S_CRLF) },
+		.chunks = (struct TfwStr *)(TfwStr []){
+			{ .data = S_200_PART_01, .len = SLEN(S_200_PART_01) },
+			{ .data = *this_cpu_ptr(&g_buf), .len = SLEN(S_V_DATE) },
+			{ .data = S_200_PART_02, .len = SLEN(S_200_PART_02) },
+			{ .data = S_CRLF, .len = SLEN(S_CRLF) },
 		},
 		.len = SLEN(S_200_PART_01 S_V_DATE S_200_PART_02 S_CRLF),
-		.flags = 4 << TFW_STR_CN_SHIFT
+		.flags = 4
```

Comment:
This is wrong. This should be `chunknum`.
Please check and change all other places that are similar to this one.